### DISCUSSION TITLE
URL 去重，get请求，url的参数不同会视为同一URL

### DISCUSSION THREAD

**Discussion by oyjcmyn** [Boris-code/feapder#206] [Ideas]
去重的URL是这个，https://xxxx.com/go ，如果 再用https://xxxx.com/go?key=aaa 和 https://xxxx.com/go?key=bbb  请求都会表示是重复URL，有办法设置不是重复么。而我需要是不想在请求 https://xxxx.com/go?key=aaa时设置filter_repeat=False。

**Comment by Boris-code** [comment]
发下你的代码，理论上这就是不重复的才对
